docs/make.jl

```diff
@@ -1,4 +1,9 @@
-using Documenter, FlameGraphs
+using Documenter, FlameGraphs, FileIO
+
+data, lidict = load(joinpath("src", "assets", "profile_ttfp.jlprof"))
+g = flamegraph(data; lidict=lidict)
+pix = flamepixels(g)
+save(joinpath("src", "assets", "ttfp.png"), reverse(pix'; dims=1))
```

Comment:
What about embedding the image generation codes into "index.md"?
<pre>
```julia
using Plots, Profile, FlameGraphs, Colors
@profile plot(rand(5))    # "time to first plot"
g = flamegraph(C=true)
```
```@setup ttfp
using FileIO, FlameGraphs, Colors
data, lidict = load(joinpath("assets", "profile_ttfp.jlprof"))
g = flamegraph(data; lidict=lidict)
```
```@repl ttfp
img = flamepixels(g);
save(joinpath("assets", "ttfp.png"), reverse(img'; dims=1)) # hide
```
we might get something like this:

![default_colors](assets/ttfp.png)

The following is an example of a customized coloration:
```@repl ttfp
fcolor = FlameColors(colorbg=colorant"#444", colorfont=colorant"azure",
                     colorsrt=colorant"lavender", colorsgc=colorant"mintcream");
img = flamepixels(fcolor, g);
save(joinpath("assets", "customized_ttfp.png"), reverse(img'; dims=1)) # hide
```
![customized_colors](assets/customized_ttfp.png)

An alternative is to color frames by their category:

```@repl ttfp
img = flamepixels(StackFrameCategory(), g);
```
in which case we might get something like this:

![sfc_colors](assets/categorize_ttfp.png)
</pre>

We might be able to make the `profile_test (n)` examples the doctests as well.